src/admin/containers/labels.js

```diff
@@ -33,18 +32,25 @@ const lines =
   , 'address3'
   , 'address4'
   , 'postcode'
-  , 'county'
+  , 'deliverer'
   ]
 
 
 const font_style = text =>
   ({ style: { fontSize: Math.min(12, 300 / text[0].length) + 'pt' } })
 
+
+const add_deliverer = arr =>
```

Comment:
Thanks rafe

On Wed, Jul 27, 2016 at 11:07 AM, Rafe <notifications@github.com> wrote:

> In src/admin/containers/labels.js
> <https://github.com/foundersandcoders/sail-back/pull/297#discussion_r72413564>
> :
>
> >    ]
> >
> >
> >  const font_style = text =>
> >    ({ style: { fontSize: Math.min(12, 300 / text[0].length) + 'pt' } })
> >
> > +
> > +const add_deliverer = arr =>
>
> The pointfree version would look something like
>
> const add_deliver = flip(chain)(flip(over)(defaultTo('POST')),
> compose(lensIndex, add(-1), length))
>
> I think it's more declarative the way I wrote. I like to avoid messing
> with indices -- particularly negative indices -- as much as possible. You
> also don't duplicate the array twice, as one does with slicing and
> spreading, but that's less important.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/foundersandcoders/sail-back/pull/297/files/e13ecf794a9b1002033929a4a93b5ec5deda5274#r72413564>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMDYNY6hVW8z9Q_k0M7QeJ1kkcOan7psks5qZy35gaJpZM4JVUvx>
> .
>